helpers/calendar/ratios_and_utilization.py

```diff
@@ -159,27 +159,39 @@ def percentage_formater(self, portion, total):
         except ZeroDivisionError:
             return 0
 
-    def get_bookings_analytics_count(self, query, start, end):
+    def get_bookings_analytics_count(self, query, start, end, room_id=None):
         results = []
-        start_date, day_after_end_date = CommonAnalytics.convert_dates(self, start, end)  # noqa E501
-        start_dt = dateutil.parser.parse(start_date)
-        end_dt = dateutil.parser.parse(day_after_end_date)
-        number_of_days = (end_dt - start_dt).days
-
+        start_date, day_after_end_date = CommonAnalytics.convert_dates(
+            self, start, end)
+        formated_start_date = dateutil.parser.parse(start_date)
+        formated_end_date = dateutil.parser.parse(day_after_end_date)
+        number_of_days = (formated_end_date - formated_start_date).days
+
+        exact_room = RoomModel.query.filter_by(
+            id=room_id).first() if room_id else None
```

Comment:
Kindly clarify why this line of code `if room_id else None` is added here, same to line 173 and 185. If it is needed, make it more readable.

Reply:
@Bonifase `room_id` is an optional argument when making a query in this case. so for instances when it is not provided `exact_room` is assigned a value of none. To make this more readable/understandable I could add a comment.